package.json

```diff
@@ -1,22 +1,19 @@
 {
-  "devDependencies": {
-    "autoprefixer": "^6.0.3",
-    "babel-core": "^6.24.0",
-    "babel-loader": "^6.4.1",
-    "babel-preset-es2015-ie": "^6.7.0",
-    "barba.js": "^1.0.0",
-    "gulp": "^3.9.0",
-    "gulp-clean": "^0.3.2",
-    "gulp-postcss": "^6.0.1",
-    "gulp-rev": "^7.1.2",
-    "gulp-sass": "^2.0.4",
-    "gulp-sourcemaps": "^1.6.0",
-    "gulp-webpack": "^1.5.0",
-    "trace.js": "^2.0.0",
-    "webpack": "^2.4.1"
-  },
-  "dependencies": {
-    "magnific-popup": "^1.1.0",
-    "raf": "^3.3.0"
-  }
+    "scripts": {
+        "build": "node ./node_modules/webpack/bin/webpack.js --config webpack.config.js"
+    },
+    "dependencies": {
+        "@wearejust/trace": "^2.0.5",
+        "autoprefixer": "^6.0.3",
+        "babel-core": "^6.24.0",
+        "babel-loader": "^6.4.1",
+        "babel-preset-es2015-ie": "^6.7.0",
+        "magnific-popup": "^1.1.0",
+        "node-sass": "^4.5.2",
+        "postcss-loader": "^2.0.5",
+        "sass-loader": "^6.0.5",
+        "style-loader": "^0.17.0",
+        "trace.js": "^2.0.0",
+        "webpack": "^2.4.1"
+    }
 }
```

Comment:
No devDependencies?

Reply:
One other thing: both wearejust/trace and trace.js can be removed from the dependencies.

Reply:
Also from devDependencies? I quess ```"@wearejust/trace": "^2.0.5",``` must be in there right?

Reply:
Sure, can be. Looks good to me :)